src/analytics/analyticsEvents.ts

```diff
@@ -295,24 +296,46 @@ export class AnalyticsEvents {
         'https://synapse.curiouscontent.org/'
       );
     }
-
-    logEvent(AnalyticsEvents.gana, 'completed', {
-      type: 'completed',
-      clUserId: AnalyticsEvents.uuid,
-      userSource: AnalyticsEvents.userSource,
-      app: AnalyticsEvents.getAppTypeFromDataURL(AnalyticsEvents.dataURL),
-      lang: AnalyticsEvents.getAppLanguageFromDataURL(AnalyticsEvents.dataURL),
-      latLong: AnalyticsEvents.joinLatLong(AnalyticsEvents.clat, AnalyticsEvents.clon),
-      // city: city,
-      // region: region,
-      // country: country,
-      score: score,
-      maxScore: maxScore,
-      basalBucket: basalBucketID,
-      ceilingBucket: ceilingBucketID,
-      appVersion: AnalyticsEvents.appVersion,
-      contentVersion: AnalyticsEvents.contentVersion,
-    });
+    if (isItLastAssessment()) {
+      console.log(">>>>>>>>>>>>>>>>SAfasfasfaf");
+      logEvent(AnalyticsEvents.gana, 'completed', {
+        type: 'completed',
+        clUserId: AnalyticsEvents.uuid,
+        userSource: AnalyticsEvents.userSource,
+        app: AnalyticsEvents.getAppTypeFromDataURL(AnalyticsEvents.dataURL),
+        lang: AnalyticsEvents.getAppLanguageFromDataURL(AnalyticsEvents.dataURL),
+        latLong: AnalyticsEvents.joinLatLong(AnalyticsEvents.clat, AnalyticsEvents.clon),
+        // city: city,
+        // region: region,
+        // country: country,
+        score: score,
+        maxScore: maxScore,
+        basalBucket: basalBucketID,
+        ceilingBucket: ceilingBucketID,
+        appVersion: AnalyticsEvents.appVersion,
+        contentVersion: AnalyticsEvents.contentVersion,
+        nextAssessment: 'Null',
+      });
+    } else {
+
+      logEvent(AnalyticsEvents.gana, 'completed', {
+        type: 'completed',
+        clUserId: AnalyticsEvents.uuid,
+        userSource: AnalyticsEvents.userSource,
+        app: AnalyticsEvents.getAppTypeFromDataURL(AnalyticsEvents.dataURL),
+        lang: AnalyticsEvents.getAppLanguageFromDataURL(AnalyticsEvents.dataURL),
+        latLong: AnalyticsEvents.joinLatLong(AnalyticsEvents.clat, AnalyticsEvents.clon),
+        // city: city,
+        // region: region,
+        // country: country,
+        score: score,
+        maxScore: maxScore,
+        basalBucket: basalBucketID,
+        ceilingBucket: ceilingBucketID,
+        appVersion: AnalyticsEvents.appVersion,
+        contentVersion: AnalyticsEvents.contentVersion,
+      });
+    }
```

Comment:
Instead of having 2 similar blocks we can just send the nextAssessment NULL or other value, which we could possibly provide from the Synapse? Like the next assessment? hausa-sight-words as an example?

also too much code duplication here